ui/raidboss/data/05-shb/eureka/zadnor.js

```diff
@@ -81,30 +133,1155 @@ export default {
+      condition: (data) => data.ce === 'shadow',
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Shadow Stormcall',
+      netRegex: NetRegexes.startsUsing({ source: 'Alkonost', id: '5E39', capture: false }),
+      condition: (data) => data.ce === 'shadow',
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Follow Slow Orb',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Shadow Stormcall Away',
+      netRegex: NetRegexes.startsUsing({ source: 'Alkonost', id: '5E39', capture: false }),
+      condition: (data) => data.ce === 'shadow',
+      delaySeconds: 15,
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Away From Orb',
+        },
+      },
+    },
     // ***** A Familiar Face *****
+    {
+      id: 'Zadnor Face Ancient Quake IV',
+      netRegex: NetRegexes.startsUsing({ source: '4th-Make Hashmal', id: '5D14', capture: false }),
+      condition: (data) => data.ce === 'face',
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Face Rock Cutter',
+      netRegex: NetRegexes.startsUsing({ source: '4th-Make Hashmal', id: '5D13' }),
+      condition: tankBusterOnParty('face'),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Zadnor Face Extreme Edge Left',
+      netRegex: NetRegexes.startsUsing({ source: 'Phantom Hashmal', id: '5D0E', capture: false }),
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Find Phantom; Dodge Left',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Face Extreme Edge Right',
+      netRegex: NetRegexes.startsUsing({ source: 'Phantom Hashmal', id: '5D0D', capture: false }),
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Find Phantom; Dodge Right',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Face Hammer Round',
+      netRegex: NetRegexes.ability({ source: '4th-Make Hashmal', id: '5D10', capture: false }),
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Away From Hammer; Rotate Outside',
+        },
+      },
+    },
     // ***** Looks to Die For *****
+    {
+      id: 'Zadnor Looks Forelash',
+      netRegex: NetRegexes.startsUsing({ source: 'Ayida', id: '5DA9', capture: false }),
+      condition: (data) => data.ce === 'looks',
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'Zadnor Looks Backlash',
+      netRegex: NetRegexes.startsUsing({ source: 'Ayida', id: '5DAA', capture: false }),
+      condition: (data) => data.ce === 'looks',
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          // TODO: should this be a response/output?
```

Comment:
This exists obviously in DR(S) as well, but Varis has "Go Front". Does it make sense to consolidate these?

Reply:
Yeah.  I think "Get In Front" reads a little bit better even if it's longer? Probably Varis should be switched to this (in some other PR).